python/setup.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+from setuptools import setup
+from os import path
+
+PACKAGE_NAME = "ccf"
+
+path_here = path.abspath(path.dirname(__file__))
+utilities_path = "tests"
+
+with open(
+    path.join(path_here, "README.md"), encoding="utf-8"
+) as f:
+    long_description = f.read()
+
+setup(
+    name=PACKAGE_NAME,
+    version="0.11.7",
+    description="Set of tools and utilities for the Confidential Consortium Framework (CCF)",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/microsoft/CCF",
```

Comment:
Nit: Should this point to the `CCF/python` folder rather than the repo homepage? I don't know if we have any other examples of "Python tools as a subfolder" to base this on.